Cargo.toml

```diff
@@ -26,10 +26,22 @@ itertools = "0.9"
 [features]
 # Select the LLVM version to be compatible with.
 # You _must_ enable exactly one of the following features.
-llvm-8 = ["llvm-sys-80"]
-llvm-9 = ["llvm-sys-90"]
-llvm-10 = ["llvm-sys-100"]
-llvm-11 = ["llvm-sys-110"]
+llvm-8 = ["llvm-sys-80", "llvm-8-or-lower"]
+llvm-9 = ["llvm-sys-90", "llvm-9-or-lower", "llvm-9-or-greater"]
+llvm-10 = ["llvm-sys-100", "llvm-10-or-lower", "llvm-10-or-greater"]
+llvm-11 = ["llvm-sys-110", "llvm-11-or-lower", "llvm-11-or-greater"]
+
+# For convenience we set a number of configuration options to avoid
+# checking complex combinations of features all the time.
+llvm-8-or-greater = [] # At least version 9
```

Comment:
I'm confused by the comment "At least version 9".  I'm fine with removing this comment (also on the next line)?